sphinx/source/docs/user_guide/server.rst

```diff
@@ -34,7 +34,9 @@ browser would provide further powerful capabilities:
   plots in the browser
 * use periodic, timeout, and asynchronous callbacks to drive streaming updates
 
-**This is where the Bokeh server comes into play.**
+**This is where the Bokeh server comes into play. Its primary purpose is to
+synchronize data between the underlying Python environment and the BokehJS
+library running in the browser.**
```

Comment:
Suggest this:
```
This is where the Bokeh server comes into play:

**The primary purpose of the Bokeh server is to synchronize data
between the underlying Python environment and the BokehJS 
library running in the browser.**
```

It probably seems like I am harping on this, but all my experience since 2016 points me towards having one clear sentence that stands alone.